docs/sphinx/source/contributing/style_guide.rst

```diff
@@ -74,9 +74,8 @@ status box at the bottom of the pull request.
 
 References
 ----------
-All pertinent information within a docstring should include a proper reference.
-In pvlib-python, we are transitioning to a standardised referencing system. We
-encourage using the `IEEE style <https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf>`_
+pvlib-python is transitioning to a standard reference format. We encourage
```

Comment:
Re-reading this, we say "standard" reference format but then "encourage" use of IEEE, leaving me unsure if the IEEE style is the standard for pvlib.

If so, then "pvlib-python is transitioning to the IEEE reference style" and "New code should use the IEEE style" would be more clear.

If we are accepting other formats, then "encourage" is OK but replace "standard" with "more consistently formatted references" I'm not in favor of this idea and I thought we agreed that IEEE is preferred.

Reply:
There should only be one correct way. Let's require IEEE for new contributions

Reply:
All good points

> I'm not in favor of this idea and I thought we agreed that IEEE is preferred.

Me neither, and yes we did

I was thinking about [this comment](https://github.com/pvlib/pvlib-python/issues/2202#issuecomment-2346879058) (the recommend not insist bit) when I used the words recommend and encourage.

I agree the current wording is unclear and should be improved though.

> "pvlib-python is transitioning to the IEEE reference style" and "New code should use the IEEE style"

I am okay with the above.

I might slightly prefer: "pvlib-python is transitioning to a consistent reference style. New code should use the IEEE style."
Possibility: "new contributions" instead of "new code"? Replace one or both of the "style" with "format"?

Reply:
> "pvlib-python is transitioning to the IEEE reference style, which should be used in all new contributions"

+1

> "new contributions" instead of "new code"

Contributions is better